net/portmapper/portmapper.go

```diff
@@ -167,11 +189,11 @@ func (c *Client) gatewayAndSelfIP() (gw, myIP netaddr.IP, ok bool) {
 }
 
 func (c *Client) invalidateMappingsLocked(releaseOld bool) {
-	if c.pmpMapping != nil {
+	if c.mapping != nil {
 		if releaseOld {
-			c.pmpMapping.release()
+			c.mapping.release(context.Background())
```

Comment:
do we want some timeout on the context here?

Reply:
I think we don't particularly care if this fails, maybe I should just fire this off in a goroutine with some deadline?